projects/storefrontlib/src/cms-components/asm/customer-emulation/customer-emulation.component.html

```diff
@@ -0,0 +1,38 @@
+<ng-container *ngIf="customer$ | async as customer">
```

Comment:
maybe remove this and subscribe through ts instead of template to get your userdetails, and just handle the unsubscription. Reason for this is in your asm-ui.html, you're already checking if customers exist or not, else don't display

Reply:
Ok, I changed it.